docs/book/src/reference/jobs.md

```diff
@@ -43,6 +43,13 @@ GitHub Presubmit Workflows:
 * release (run on tags)
   * Creates a GitHub release with release notes for the tag.
 
+
+GitHub Weekly Workflows:
+* golangci-lint: golangci/golangci-lint-action
+  * Weekly check all Markdown links
+* scan-images: aquasecurity/trivy-action
+  * Scan all images for vulnerabilities. Can be run locally via `make verify-container-images`
```

Comment:
We now have two diverging implementations for how to scan, what about just calling the make target in the GitHub action?

This should make it easier to keep implementations (and trivy versions) in sync